### PR TITLE
WFLY-3171 Increase default state transfer timeout

### PR DESCRIPTION
**Pull request by rhusar**
Jira
https://issues.jboss.org/browse/WFLY-3171

### CODE REVIEW

File: clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/StateTransferResourceDefinition.java [hunk 1]
```diff
@@ -92,8 +95,10 @@ static SimpleAttributeDefinitionBuilder createBuilder(String name, ModelType typ
     }
 
     static void buildTransformation(ModelVersion version, ResourceTransformationDescriptionBuilder parent) {
+        ResourceTransformationDescriptionBuilder builder = InfinispanModel.VERSION_4_0_0.requiresTransformation(version) ? parent.addChildRedirection(PATH, LEGACY_PATH) : parent.addChildResource(PATH);
+
         if (InfinispanModel.VERSION_4_0_0.requiresTransformation(version)) {
-            parent.addChildRedirection(PATH, LEGACY_PATH);
```
Comment (kabir) [inline clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/StateTransferResourceDefinition.java]: Is this not needed?

Comment (rhusar) [inline clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/StateTransferResourceDefinition.java]: Yes, it is, see 2 lines above. :-)

**Comment by rhusar** [comment]
@pferraro Any objections?

**Comment by pferraro** [comment]
:+1: Looks good.

**Comment by pferraro** [comment]
Actually, we need to change the default values for all versions of the xsd.

**Comment by rhusar** [comment]
Done now, rerunning the tests.

**Pull request merged**